vars/common_vars.yml.template

```diff
@@ -5,8 +5,6 @@ users:
     uid: 1100
     # Password created with `mkpasswd --method=SHA-512 pass1234`
     password: $6$6/6NkDn01w$6ld/sdc/B42cVtSNFQTQLlNPYOqin8m4kluy.QbvtLraHLliTs5QaPIYJcKyYr7KKhsY2znFZLSC2YzdROKwZ.
-    # Also add roles/common/files/<username>.key and roles/common/files/<username>.key.pub
```

Comment:
What happened with the users key pair? Are they not needed for users to be able to ssh between nodes?

Reply:
It was moved to **files/** in the root directory so the user does not change or add anything to the **roles/**. Hence, this comment was missing leading. Plus the information to create such key is in the README which the user will read before reaching this point.